src/aspire/source/__init__.py

```diff
@@ -64,22 +105,41 @@ def __init__(self, L, n, dtype='double', metadata=None):
             self._metadata = pd.DataFrame([], index=pd.RangeIndex(self.n))
         else:
             self._metadata = metadata
+            self._rotations = R.from_euler('ZYZ', self.get_metadata(['_rlnAngleRot', '_rlnAngleTilt', '_rlnAnglePsi']), degrees=True)
+
+        self.model_pipeline = Pipeline(xforms=None, memory=memory)
+        self.generation_pipeline = Pipeline(xforms=None, memory=memory)
```

Comment:
Why are these not initialized with the values from the metadata fields?

Reply:
`model_pipeline` obliterated. Vaporized.